runtime/js-compute-runtime/js-compute-builtins.h

```diff
@@ -32,6 +32,8 @@ const JSErrorFormatString js_ErrorFormatString[JSErrNum_Limit] = {
 
 #include "host_interface/host_api.h"
 
+#define RUNTIME_VERSION "3.13.2-dev"
```

Comment:
We talked about this briefly earlier, but it would be great to figure out how to get `release-please` to keep this in sync.

Reply:
Will follow up as soon as I've worked out the release process.